magento/pwa-studio
# [feature]: My Account: Account Information

## Description

As a shopper, I want to see all associated information for my account so that I can view/edit/delete data (contact info, billing, address, etc.) saved to my profile.

## Related Issue





Closes #2668.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Login to Venia site with your account registered.
2. Navigate to Account information page.
3. Make sure you can see all content of Account information page.
4. Change the viewport to mobile and verify rendered to mobile mocks
5. Change your first/last/email. Click "save". When the modal closes, the data displayed should match what you entered. Also, the header name should match the first name you just used.
6. Enter a new password that is same as your old password and click "save". It should error.
7. Enter a new password that is different from your old password and click "save". It should work.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
